tests/acceptance/run.sh

```diff
@@ -1187,5 +1199,7 @@ fi
 if [ -n "${EXPECTED_FAILURES_FILE}" ]
 then
 	echo "runsh: Exit code after checking expected failures: ${FINAL_EXIT_STATUS}"
+	echo "runsh: Total failed scenarios throughout the test run:"
+	printf "%s\n" "${UNEXPECTED_FAILED_SCENARIO_PATHS[@]}"
```

Comment:
There might be no unexpected failures. This could echo and printf could go inside an `if`  statement, maybe something like:
```suggestion
if [ "${UNEXPECTED_FAILURE}" = true ]
then
	echo "runsh: Total failed scenarios throughout the test run:"
	printf "%s\n" "${UNEXPECTED_FAILED_SCENARIO_PATHS[@]}"
else
	echo "runsh: There were no unexpected failures."
fi
```